Sample Data/Custom/CyberArkEPM_CL.json

```diff
@@ -0,0 +1,414 @@
+[
+    {
+       "fileQualifier":"-7364030719992865801",
+       "lastEventDisplayName":"Microsoft OneDriveFile Co-Authoring Executable (FileCoAuth.exe)",
+       "hash":"SHA1##420BE0941CEB6B0781A93D6ED81B1C80E733A5FF",
+       "publisher":"Microsoft Corporation",
+       "totalEvents":7,
+       "applicationType":"Executable",
+       "eventType":"Launch",
+       "lastEventSourceType":"Updater",
+       "lastEventSourceName":"Updater (Windows Update)",
+       "lastEventDate":"2022-03-10T13:32:03.206Z",
+       "lastEventUserName":"DESKTOP-S4J3FEM\\User1",
+       "lastEventJustification":"",
+       "threatDetectionAction":"",
+       "lastEventFileName":"FileCoAuth.exe",
+       "affectedComputers":1,
+       "affectedUsers":1,
+       "firstEventDate":"2022-03-10T12:32:02.642Z",
+       "firstEventUserName":"DESKTOP-S4J3FEM\\Adam",
+       "fileSize":921504,
+       "CLSID":"",
+       "mimeType":"",
+       "url":"",
+       "appPackageDisplayName":"",
+       "agentId":"f67fa0d1-300a-45c0-b199-8d74821efd29",
+       "fileLocation":"",
+       "skippedCount":0,
+       "skipped":false,
+       "aggregatedBy":"420BE0941CEB6B0781A93D6ED81B1C80E733A5FF,2048",
+       "deceptionType":0,
+       "defenceActionId":0,
+       "lastAgentId":"f67fa0d1-300a-45c0-b199-8d74821efd29",
+       "event_type":"aggregated_events"
+    },
+    {
+       "policyName":null,
+       "hash":"420BE0941CEB6B0781A93D6ED81B1C80E733A5FF",
+       "publisher":"Microsoft Corporation",
+       "eventType":"Launch",
+       "sourceType":"Updater",
+       "sourceName":"Updater (Windows Update)",
+       "lastEventDate":"2022-03-10T13:32:03.206Z",
+       "firstEventDate":"2022-03-10T13:32:03.206Z",
+       "userName":"DESKTOP-S4J3FEM\\Adam",
+       "justification":null,
+       "fileName":"FileCoAuth.exe",
+       "originalFileName":"FileCoAuth.exe",
+       "fileSize":921504,
+       "threatProtectionAction":"ALL",
+       "packageName":"Microsoft OneDrive (OneDriveSetup.exe)",
+       "company":"Microsoft Corporation",
+       "filePath":"C:\\Users\\Adam\\AppData\\Local\\Microsoft\\OneDrive\\22.033.0213.0002\\FileCoAuth.exe",
+       "fileDescription":"Microsoft OneDriveFile Co-Authoring Executable",
+       "productName":"Microsoft OneDrive",
+       "productVersion":"22.33.213.2",
+       "bundleName":null,
+       "bundleVersion":null,
+       "bundleId":null,
+       "applicationSubType":null,
+       "fileVersion":"22.33.213.2",
+       "modificationTime":"2022-03-09T06:15:26.532Z",
+       "userIsAdmin":false,
+       "agentEventCount":1,
+       "skippedCount":0,
+       "accessAction":"false",
+       "accessTargetType":"Internet",
+       "accessTargetName":null,
+       "processCommandLine":"-Embedding",
```

Comment:
Hey @averbn , in a parser it is given as guid( _g) for "processCommandLine" but in Sample Data the value is string, please use GUID in sample data or use "_s" for processCommandLine in parser, please check and update, thanks.
![image](https://user-images.githubusercontent.com/94349919/158120897-34612a62-b3b4-4b01-abce-99d3c2325508.png)

Reply:
I don`t understand what I should change: we have two fields processCommandLine_g and processCommandLine_s. One for guide and second for process cmd. I think both should be with string Type. I see that the field processCommandLine_g is missing in the examples - I will add it